src/data/community-events.json

```
@@ -53,6 +53,15 @@
     "startDate": "2023-05-26",
     "endDate": "2023-05-28"
   },
+  {
+    "title": "ETH Belgrade 2023",
+    "to": "https://belgradeblockchainweek.com/#eth-belgrade",
+    "sponsor": null,
+    "location": "Belgrade, Serbia",
+    "description": "The CEE region's premier ETH conference dedicated to the Ethereum and the Ethereum ecosystem.",
```

```suggestion
    "description": "Belgrade Blockchain Week is week-long in-person gathering of the greatest Web3 minds and professionals",
```

Pulling something a little less marketing from the website